Dockerfile

```diff
@@ -28,14 +28,10 @@ COPY controllers/ controllers/
 RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o /src/${PROJECT_NAME} \
     -ldflags "-X ${REPO_PATH}/pkg/version.Version=${VERSION} -X ${REPO_PATH}/pkg/version.GitSHA=${GIT_SHA}" main.go
 
-FROM ${DOCKER_REGISTRY:+$DOCKER_REGISTRY/}alpine:${ALPINE_VERSION} AS final
-
+FROM gcr.io/distroless/static:nonroot AS final
```

Comment:
Currently available images are: https://github.com/GoogleContainerTools/distroless#what-images-are-available

That is, maybe we should use `gcr.io/distroless/static-debian11` directly instead of `gcr.io/distroless/static`. I think `static` is a link to the latest build `static-debian11`, but we should use the latest build explicitly, so that we have a reproducible build and so we don't auto-upgrade images when the `static` link changes.

Reply:
Done